### PR TITLE
Feature: Updated person page

### PR DESCRIPTION
**Pull request by pyrello**
Relates to #2320, although I'm not sure if it resolves it.

Adds the updated design of the person page to the default platform from Admissions.

# How to test

Mainly this is a matter of comparing the new versions of pages with the existing versions of those pages to make sure that all the same information is represented.

* `yarn workspace uids_base build`
* V3: `blt ds && drush @default.local uli people/herky-hawk`
* ^ Compare to: https://demo.sitenow.uiowa.edu/people/herky-hawk
* Admissions: `blt ds --site=admissions.uiowa.edu && drush @admissions.local uli people/amanda-cvitko-quinteiro`
* ^ Compare to: https://admissions.prod.drupal.uiowa.edu/people/amanda-cvitko-quinteiro - **Note:** These should look exactly the same, except that the background image behind the email address has been removed.
* V2: `blt ds --site=hawkeyemarchingband.uiowa.edu && drush @hawkeyemarchingband.local uli people/michael-janssen`
* ^ Compare to: https://hawkeyemarchingband.uiowa.edu/people/michael-janssen
* Person extended: `blt ds --site=informatics.grad.uiowa.edu && drush @gradinformatics.local uli research-faculty/tong-wang`
* ^ Compare to: https://informatics.grad.uiowa.edu/research-faculty/tong-wang
* Immunology: `blt ds --site=immuno.grad.uiowa.edu && drush @gradimmuno.local uli people/priya-issuree`
* ^ Compare to: https://immuno.grad.uiowa.edu/people/priya-issuree
* Visit http://gradimmuno.local.drupal.uiowa.edu/people/samuel-connell
* ^ Compare to: https://immuno.grad.uiowa.edu/people/samuel-connell
* Visit http//gradimmuno.local.drupal.uiowa.edu/people/rebecca-benson
* ^ Compare to: https://immuno.grad.uiowa.edu/people/rebecca-benson
* Check other examples you are aware of or at random!

### CODE REVIEW

**Comment by pyrello** [comment]
Remaining todo:
- [x] Compare default `core.entity_view_display.node.person.default.yml` against Admissions.
- [x] ~Update https://github.com/uiowa/uiowa/blob/main/config/grad.uiowa.edu/core.entity_view_display.node.person.default.yml~ - Going to handle this in #2621 
- [x] Update https://github.com/uiowa/uiowa/blob/main/config/immuno.grad.uiowa.edu/core.entity_view_display.node.person.default.yml
- [x] Update https://github.com/uiowa/uiowa/blob/main/config/features/person_extended/core.entity_view_display.node.person.default.yml
- [x] Potentially remove the `node--person--full` template.

File: docroot/themes/custom/uids_base/templates/views/views-view-unformatted--person-card.html.twig [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    ]
+    %}
+
+    {% set has_body = false %}
+
+    {% set card_body_fields = ['card_subtitle', 'card_tag', 'person_website', 'person_education', 'person_hometown', 'person_contact'] %}
+
+    {% for check in card_body_fields %}
+      {% if person_card_view[check]|striptags|trim is not empty %}
+        {{ check }}
```
Comment (cory-skeers) [inline docroot/themes/custom/uids_base/templates/views/views-view-unformatted--person-card.html.twig]: Is this still needed for the last todo items, or can it be taken out? (Specifically the `{{ check }}`

Comment (pyrello) [reply docroot/themes/custom/uids_base/templates/views/views-view-unformatted--person-card.html.twig]: It is stripped in a commit that I have not pushed yet.

**Comment by pyrello** [comment]
Updated testing instructions to give examples to compare pages against and to add more cases affected by this PR.

**Comment by pyrello** [comment]
I just realized that we have not accounted for credentials in this. Where are these supposed to go @bspeare?

**Comment by pyrello** [comment]
I have added the credentials to the end of the person's name (page title).

**Comment by cory-skeers** [comment]
It looks like the Dissertation Title (like on https://gradimmuno.local.drupal.uiowa.edu/people/rebecca-benson) is currently wrapped in an `<h2>`, but not in these updates--not sure if that was an intentional change at some point?

Otherwise I've gone through the examples listed and things are looking really good!

**Comment by pyrello** [comment]
It was not an intentional change, just the effect of removing the person template. I added a field template to handle this.

**Review by cory-skeers — APPROVED** [review]
From the testing I've done, this looks good to go 👍

**Pull request merged**